### ISSUE TITLE
add show() method (analogous to matplotlib)

### ISSUE THREAD

**Issue by drs251** [holoviz/holoviews#2481]
For workflows which don't involve Jupyter notebooks, it would be really nice to have a show() method, that opens a new GUI window (like matplotlib does) or browser tab (perhaps more appropriate for bokeh). The syntax could be something like this:

```python
import holoviews as hv
my_curve = hv.Curve(my_data)
my_curve.show(backend="matplotlib", dest="window")
my_curve.show(backend="bokeh", dest="browser")
```

**Comment by philippjfr** [comment]
Unlikely to add it as a method on objects but I wouldn't be opposed to a top-level ``hv.show``.

**Comment by jlstevens** [comment]
I think ``h.show()`` is reasonable for work at the interactive prompt (as suggested by @drs251 ) but I really don't want to see it used in other contexts (notebooks, bokeh server etc).

**Comment by Dr-Irv** [comment]
So I was looking at how to have a `BokehRenderer.show()`, and then found that `BokehRenderer.app(obj, show=True)` does the job for a single object. Before discovering `BokehRenderer.app()`, I had implemented something for my own use which would handle opening multiple tabs when passed a list of objects, and in each tab, I added a Bokeh button that says "Close tab", and when the last button is hit, the server application is stopped.  I could integrate this into the  `BokehRenderer.app()` implementation, but I wasn't sure if that method was used for anything else.
